packages/atomic/src/decorators/light-dom.ts

```diff
@@ -0,0 +1,37 @@
+import {CSSResult} from 'lit';
+
+export const injectStylesForNoShadowDOM = <
+  T extends {
+    styles: CSSResult;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    new (...args: any[]): any;
+  },
+>(
+  Base: T
+) => {
+  return class extends Base {
+    createRenderRoot() {
+      return this;
+    }
+
+    public connectedCallback() {
+      super.connectedCallback();
+      this.injectStyles();
+    }
+
+    injectStyles() {
+      const parent = this.getRootNode();
+      const styleSheet = Base.styles?.styleSheet;
```

Comment:
IIRC `Base.styles` could be an array as well do we handle this case?

Reply:
Yes, but for the components that require this decorator, the CSS should be quite simple and can be added directly using the CSSResult type. If we need to support style array, we can update the decorator accordingly but I feel like this is unnecessary complexity for the time being.